starter-kits/credential-registry/server/tob-api/icat_cbs/views.py

```diff
@@ -155,9 +148,9 @@ def handle_credentials(state, message):
 
             # Instruct the agent to store the credential in wallet
             resp = requests.post(
-                f"{AGENT_ADMIN_URL}/credential_exchange/{credential_exchange_id}/store",
+                f"{django.conf.settings.AGENT_ADMIN_URL}/credential_exchange/{credential_exchange_id}/store",
```

Comment:
The import for the settings package seems to be missing. I suggest using `from django.conf import settings` rather than importing the full django package.